.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
`CACHE_DIR` is very general, maybe something more specific `MINK_DEBUG_CACHE_DIR`?